assets/terraform-modules/bare-metal/flatcar-linux/kubernetes/ssh.tf

```diff
@@ -74,7 +77,7 @@ resource "null_resource" "copy-controller-secrets" {
       "sudo mv etcd-peer.key /etc/ssl/etcd/etcd/peer.key",
       "sudo chown -R etcd:etcd /etc/ssl/etcd",
       "sudo chmod -R 500 /etc/ssl/etcd",
-      "sudo mv $HOME/kubeconfig /etc/kubernetes/kubeconfig",
+      "sudo systemctl restart etcd",
```

Comment:
Perhaps we can commit etcd related changes and kubelet changes separately, so diff is clearer? Some context why this is needed would be nice too :)

Reply:
Will split.